src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -190,6 +193,7 @@ export class CalcitePickListItem {
   @Method()
   async setFocus(): Promise<void> {
     this.focusEl?.focus();
+    this.focusDiv?.focus();
```

Comment:
I'm not sure it will ever focus the `focusEl` in this case. It will always focus the div. Is that correct? I'm not sure we can focus one and then the other.

Reply:
maybe it should be written so that it's clear only one of these is going to get focused?